server/etcdmain/help.go

```diff
@@ -210,6 +210,8 @@ Experimental distributed tracing:
     Distributed tracing service name, must be same across all etcd instances.
   --experimental-distributed-tracing-instance-id ''
     Distributed tracing instance ID, must be unique per each etcd instance.
+  --experimental-distributed-tracing-sampling-rate '0'
```

Comment:
optional: worth identifying 0 as disabled?

Reply:
> It defaults to 0, I wasn't sure if we should default to sample all instead of 0, but this is how the Kubernetes implementation did it so I tried to follow the same principles as there.

Yeah, this was my main question in the description, as this disables it essentially not sure but I guess you are right if we document that this makes it disabled than we should maybe not make it optional? 🤔